### PR TITLE
Add .gitlab-ci.yml

### PR DESCRIPTION
**Pull request by krichter722**
I'm aware that this might be closed pretty much immediately because the script requires maintenance and a CI is already set up, however I want to share my finding during the long process of enable GitLab CI for this project which might be useful to someone who wishes to develop on GitLab. [The build passes](https://gitlab.com/krichter/Payara/-/jobs/83585365).

### CODE REVIEW

**Comment by arjantijms** [comment]
Jenkins test please

**Comment by arjantijms** [comment]
Jenkins test please

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+main:
```
Comment (lprimak) [inline .gitlab-ci.yml]: Missing copyrights, a blurb about what this could be used for

**Comment by lprimak** [comment]
This code is pretty small, and it if could help somebody, why not?

**Comment by arjantijms** [comment]
Closed for now, if it's ready, just re-open.

**Comment by krichter722** [comment]
> Missing copyrights, a blurb about what this could be used for

Added.

**Comment by krichter722** [comment]
@arjantijms I updated the PR. I cannot re-open this PR. Can you?

**Comment by krichter722** [comment]
My force-push doesn't seem to be recognized, I'll open a new one: #4053

**Pull request closed without merging**